sdk/inc/azure/platform/az_mqtt5_mosquitto.h

```diff
@@ -131,16 +131,16 @@ typedef struct
 } az_mqtt5_property_binarydata;
 
 /**
- * @brief Initializes the MQTT 5 instance specific to Mosquitto.
+ * @brief Initializes the MQTT 5 instance specific to Eclipse Mosquitto.
  *
  * @param mqtt5 The MQTT 5 instance.
+ * @param mosquitto_handle The Mosquitto handle, can't be NULL.
```

Comment:
Can you clarify here that the parameter can't be NULL, but the underlying value of the handle can be at this point?